app/features/idl/interactive-idl/model/use-instruction-form.ts

```diff
@@ -1,8 +1,53 @@
+'use client';
+
 import type { ArgField, InstructionAccountData, InstructionData, NestedInstructionAccountsData } from '@entities/idl';
 import { type Path, type SubmitHandler, useForm, type UseFormSetValue } from 'react-hook-form';
 
 import { isRequiredArg } from '../lib/instruction-args';
 
+type ArgumentType =
+    | 'bool'
+    | 'u8'
+    | 'u16'
+    | 'u32'
+    | 'u64'
+    | 'u128'
+    | 'i8'
+    | 'i16'
+    | 'i32'
+    | 'i64'
+    | 'i128'
+    | 'f32'
+    | 'f64'
+    | 'string'
+    | 'bytes'
+    | 'publicKey'
+    | 'pubkey';
+
+/* eslint-disable sort-keys-fix/sort-keys-fix */
+const DEFAULT_VALUES_PER_TYPE: Record<ArgumentType, string> = {
```

Comment:
Let's move these constants into the `/form-prefill` directory to the same level we keep hardcoded accounts. At the moment related functionality is located in different palces